CONTRIBUTORS.txt

```diff
@@ -1,4 +1,4 @@
-# Contributions to the urllib3 project
+﻿# Contributions to the urllib3 project
```

Comment:
I dunno, but those are my details and please add them whatever works best for you.